src/components/ConnectionLayout.tsx

```diff
@@ -60,6 +60,12 @@ type ConnectionLayoutProps = {
 
     /** Name of the current connection */
     connectionName: ConnectionName;
+
+    /** Block the screen when the connection is not empty */
+    reverseConnectionEmptyCheck?: boolean;
```

Comment:
For the Token Input we want that the connection should be empty and it should be blocked when connection is not empty.  I couldn't come up with a better name.

Reply:
How about `isForEmptyConnection